### PR TITLE
Running TCK tests for Microprofile.3.0

### PR DESCRIPTION
**Pull request by tonyreigns**
Fixes: #7880

Running mpOpenTracing TCK against a server with microprofile-3.0.

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

**Comment by fmhwong** [comment]
Does this run both OT and OT REST client TCK?

**Comment by tonyreigns** [comment]
#build

File: dev/com.ibm.ws.opentracing.1.3_fat/publish/servers/OpentracingTCKServerMicroProfile/server.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--Copyright (c) 2017 IBM Corporation and others. All rights reserved. This 
+	program and the accompanying materials are made available under the terms 
+	of the Eclipse Public License v1.0 which accompanies this distribution, and 
+	is available at http://www.eclipse.org/legal/epl-v10.html Contributors: IBM 
+	Corporation - initial API and implementation -->
+<server>
+	<featureManager>
+		<feature>componenttest-1.0</feature>
+		<feature>webProfile-8.0</feature>
```
Comment (fmhwong) [inline dev/com.ibm.ws.opentracing.1.3_fat/publish/servers/OpentracingTCKServerMicroProfile/server.xml]: Do we still need `webProfile-8.0`?

**Comment by tonyreigns** [comment]
#build

**Pull request merged**